spec/coinbase/user_spec.rb

```diff
@@ -19,6 +19,15 @@
     let(:create_wallet_request) { { wallet: { network_id: network_id } } }
     let(:opts) { { create_wallet_request: create_wallet_request } }
     let(:wallet_model) { Coinbase::Client::Wallet.new({ 'id': wallet_id, 'network_id': network_id }) }
+    let(:wallet_model_with_default_address) do
+      Coinbase::Client::Wallet.new(
+        {
+          'id': wallet_id,
+          'network_id': 'base-sepolia',
+          'default_address': '0xdeadbeef'
```

Comment:
default_address should be an address instance and not string?

Reply:
Thanks, fixed!